apps/prairielearn/src/ee/lib/context-parsers/documentation.ts

```diff
@@ -108,6 +116,39 @@ function cleanElementSections(elementSections: ElementSection[]) {
         }
         node.depth -= 1;
       }
+
+      if (node.type === 'table') {
```

Comment:
I would implement this as a separate transform independent of `cleanElementSections`.

Reply:
changed in 8a44bde82a93f2a00a3b9ca8b0b27a90f81c059f- was sitting in vscode, sorry